types/decimal_test.go

```diff
@@ -54,6 +54,10 @@ func (s *decimalTestSuite) mustNewDecFromStr(str string) (d sdk.Dec) {
 	return d
 }
 
+func TestApproxRoot(t *testing.T) {
+
+}
```

Comment:
I think this is an accident?

Reply:
ohh yes, this is an accident, sorry I forgot to delete this